cardano-api/src/Cardano/Api/DefaultValue.hs

```diff
@@ -0,0 +1,12 @@
+module Cardano.Api.DefaultValue
+  ( DefaultValue(..)
+  ) where
+
+class DefaultValue a where
+  defaultValue :: a
```

Comment:
Why aren't you using [`Deafult`](https://hackage.haskell.org/package/data-default-class-0.1.2.0/docs/Data-Default-Class.html#t:Default) from `data-default-class`? We make use of it extensively in ledger. Seems totally redundant to introduce an exact custom copy of a known concept . It's like redefining `Semigroup` with a different name. Very strange.

Reply:
Now using the `Default` class from `data-default-class`.